crates/re_space_view_spatial/src/ui_2d.rs

```diff
@@ -26,186 +23,108 @@ use crate::{
 
 // ---
 
-#[derive(Clone, Default, PartialEq, serde::Deserialize, serde::Serialize)]
+#[derive(Clone, Default, PartialEq, Eq, serde::Deserialize, serde::Serialize)]
 #[serde(default)]
 pub struct View2DState {
-    /// The zoom and pan state, which is either a zoom/center or `Auto` which will fill the screen
-    zoom: ZoomState2D,
+    /// The visible parts of the scene, in the coordinate space of the scene.
+    pub bounds: Option<Rect>,
```

Comment:
thinking a bit ahead I'm not sure how we'd call it on the exposed blueprint. Is `visible_bounds` more accurate? It's definitely distinct from `displayed_bounds` which can't be controlled directly right now.

Reply:
a comment on the meaning of `None` wouldn't hurt either :)